services/event.ts

```diff
@@ -0,0 +1,21 @@
+import { addDoc, collection, doc, getDoc } from 'firebase/firestore'
+import db from '../firebase'
+import Event, { EventData } from '../types/events'
+
+export async function createEvent(eventData: EventData) {
+  await addDoc(collection(db, 'events'), {
```

Comment:
will the event filter be located around here later? 

like getEventbyDate, or getEventbyDistance??

Reply:
We'll probably add some later when needed